### PR TITLE
[CWS] fix agent rules documentation README and docker workflow

### PR DESCRIPTION
**Pull request by paulcacheux**
### What does this PR do?

This PR fixes the readme and the docker workflow used to generate CWS agent rules documentation

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] The appropriate `team/..` label has been applied, if known.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by alai97 — APPROVED** [review]
Looks good for docs!

File: docs/cloud-workload-security/README.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ Make an edit to any of the files mentioned above and then, from the root of the
 If a `*.go` file in `pkg/security` has been edited you will first need to generate the `*.json` files.
 Please run:
 ```sh
-go generate ./pkg/security/...
+inv -e security-agent.cws-go-generate
 # or only the specific file
 go generate ./path/to/the/touched/file
 ```
```
Comment (spikat) [inline docs/cloud-workload-security/README.md:131]: v-- In the next line, we should also modify the
(at the root of `docs/cloud-workload-security`)
per:
(at the root of the repository)
Otherwize this leads to this error:
python3: can't open file './docs/cloud-workload-security/scripts/secl-doc-gen.py': [Errno 2] No such file or directory

Comment (spikat) [reply docs/cloud-workload-security/README.md:132]: Oh ok, I see :)

**Review by spikat — APPROVED** [review]
Looks ok to me

**Pull request merged**